test/e2e/case18_compliance_api_test.go

```diff
@@ -112,4 +110,757 @@ var _ = Describe("Test policy webhook", Label("compliance-events-api"), Ordered,
 			}, defaultTimeoutSeconds, 1).Should(Succeed())
 		})
 	})
+
+	Describe("Test POSTing Events", func() {
+		Describe("POST one valid event with including all the optional fields", func() {
+			payload := []byte(`{
+				"cluster": {
+					"name": "cluster1",
+					"cluster_id": "test1-cluster1-fake-uuid-1"
+				},
+				"parent_policy": {
```

Comment:
Do you think this would be better as a key under the `policy` map?

Reply:
Mostly I don't think it would matter much. The current scheme is kind of nice because the stuff in the `policy` basically matches the stuff in a k8s `policy` object (which does not have this proposed kind of embedding of the parent policy)